### PR TITLE
add 2020 talks by Christian Eder and Janko

### PR DESCRIPTION
**Pull request by rfourquet**
Preview link: https://rfourquet.github.io/oscar-website/talks/
@ederc : do you like to provide a pdf of your talk (file or url)? Also, I added an url for the "Workshop on Computational Algebra 2020" but there might be a better one?

### CODE REVIEW

**Comment by ederc** [comment]
You can use https://www.mathematik.uni-kl.de/~ederc/download/workshop-computational-algebra-2020.pdf as url to the talks. Everything else is fine for me. Thank you!

**Comment by rfourquet** [comment]
Thanks, I've added the pdfs. I also set the location of "Workshop on Computational Algebra 2020" to Kaiserslautern even if it was an online event, as Kaiserslautern is mentioned on the website and without adding it, there is spurious trailing comma. Please tell me if this is incorrect.

**Comment by rfourquet** [comment]
I added one ICMS talk by @jankoboehm (do you like to provide a (link to a) pdf file?
Also it has a different conference URL than the one I had found, any opitnion on which is preferable? The first URL seems more official
1.  http://icms-conference.org/2020/
2.  http://people.math.binghamton.edu/schroeter/2020ICMS/

File: _data/talks.yml [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+  conference: ICMS 2020 (virtual)
+  conference_url: http://icms-conference.org/2020/
+  location: "Braunschweig, Germany"
+  pdf_url: https://www.mathematik.uni-kl.de/~ederc/download/workshop-computational-algebra-2020.pdf
+
+- title: "Massively parallel fan traversals and applications"
+  author: Janko Boehm
+  date: "2020-07-13"
+  conference: ICMS 2020 (virtual)
+  conference_url: http://people.math.binghamton.edu/schroeter/2020ICMS/
```
Comment (rfourquet) [inline _data/talks.yml]:
```suggestion
  conference_url: http://icms-conference.org/2020/
```

**Pull request merged**